addons/mass_mailing/static/src/js/mass_mailing_design_constants.js

```diff
@@ -1,62 +1,6 @@
 /** @odoo-module alias=mass_mailing.design_constants**/
 
 export const CSS_PREFIX = '.o_mail_wrapper';
-export const DEFAULT_CSS_OBJECT = {
```

Comment:
A commit with `[REF]` in stable seems weird to me. Since you're removing stuff, I'd suggest `[REM]` but that's kinda weird as well in stable. Maybe just put it in the other commit and have the two descriptions together ? This one makes no sense without the other anyway.